src/components/dashboard/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Grid } from '@mui/material';
+import TenantSelector from 'components/shared/TenantSelector';
+import {
+    semiTransparentBackground_blue,
+    semiTransparentBackground_purple,
+    semiTransparentBackground_teal,
+} from 'context/Theme';
+import { CloudDownload, CloudUpload, DatabaseScript } from 'iconoir-react';
+import EntityStatOverview from './EntityStatOverview';
+import useMonthlyUsage from './EntityStatOverview/useMonthlyUsage';
+
+const ICON_SIZE = 12;
+
+export default function Dashboard() {
+    const { captureUsage, isLoading, materializationUsage } = useMonthlyUsage();
+
+    return (
```

Comment:
![image](https://github.com/user-attachments/assets/9ab02240-eee0-41f4-9e34-cf57ba039b6e)

I think this section should probably handle smaller screen sizes a bit better and make the cards stack vertically.